### DISCUSSION TITLE
Adding tags in TagsInput component

### DISCUSSION THREAD

**Discussion by waseem-k-08** [mantinedev/mantine#6112] [Feature requests]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.8.0

### What package has an issue?

@mantine/core

### What framework do you use?

Other, I will specify in the bug description

### In which browsers you can reproduce the issue?

All

### Describe the bug

In the TagsInput component, tags are currently only added by pressing the enter key. However, this functionality is not very customizable. Please make improvements so that tags can also be added by clicking outside of the component, pressing the tab key, and pressing the space key.

### Possible fix

Allow other key like tab, space to onChange prop.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
https://mantine.dev/core/tags-input/#split-chars

**Reply by mrclrchtr** [reply]
Thanks, but is there a way to trigger the add when clicking somewhere else (onBlur for example)?

**Reply by rtivital** [reply]
This feature will be available in 7.11.0 release and enabled by default, it can be configured with `acceptValueOnBlur` prop.

**Reply by mrclrchtr** [reply]
Thank you very much!
